airbyte-integrations/connectors/source-amazon-seller-partner/source_amazon_seller_partner/schemas/GET_ORDER_REPORT_DATA_SHIPPING.json

```diff
@@ -0,0 +1,192 @@
+      "format": "date"
+    },
+    "OrderPostedDate": {
+      "type": ["null", "string"],
+      "format": "date"
+    },
+    "BillingData": {
+      "type": ["null", "object"],
+      "additionalProperties": true,
+      "properties": {
+        "BuyerEmailAddress": {
+          "type": ["null", "string"]
+        },
+        "BuyerName": {
+          "type": ["null", "string"]
+        },
+        "BuyerPhoneNumber": {
+          "type": ["null", "string"]
+        }
+      }
+    },
+    "FulfillmentData": {
+      "type": ["null", "object"],
+      "additionalProperties": true,
+      "properties": {
+        "FulfillmentMethod": {
+          "type": ["null", "string"]
+        },
+        "FulfillmentServiceLevel": {
+          "type": ["null", "string"]
+        },
+        "Address": {
+          "type": ["null", "object"],
+          "additionalProperties": true,
+          "properties": {
+            "Name": {
+              "type": ["null", "string"]
+            },
+            "AddressFieldOne": {
+              "type": ["null", "string"]
+            },
+            "City": {
+              "type": ["null", "string"]
+            },
+            "StateOrRegion": {
+              "type": ["null", "string"]
+            },
+            "PostalCode": {
+              "type": ["null", "string"]
+            },
+            "CountryCode": {
+              "type": ["null", "string"]
+            },
+            "PhoneNumber": {
+              "type": ["null", "string"]
+            }
+          }
+        }
+      }
+    },
+    "IsBusinessOrder": {
+      "type": ["null", "string"]
+    },
+    "Item": {
+      "type": ["null", "object"],
+      "additionalProperties": true,
+      "properties": {
+        "AmazonOrderItemCode": {
+          "type": ["null", "string"]
+        },
+        "SKU": {
+          "type": ["null", "string"]
+        },
+        "Title": {
+          "type": ["null", "string"]
+        },
+        "Quantity": {
+          "type": ["null", "string"]
+        },
+        "ProductTaxCode": {
+          "type": ["null", "string"]
+        },
+        "ItemPrice": {
+          "type": ["null", "object"],
+          "additionalProperties": true,
+          "properties": {
+            "Component": {
+              "type": ["array"],
+              "items": {
+                "type": ["null", "object"]
+              },
+              "properties": {
+                "Type": {
+                  "type": ["null", "string"]
+                },
+                "Amount": {
+                  "type": ["null", "object"],
+                  "additionalProperties": true,
+                  "properties": {
+                    "currency": {
+                      "type": ["null", "string"]
+                    },
+                    "value": {
+                      "type": ["null", "string"]
+                    }
+                  }
+                }
+              }
+            }
+          }
```

Comment:
This doesn't exactly match the docs, but I'm guessing you got it from querying the api?  https://developer-docs.amazon.com/sp-api/docs/order-reports-attributes#get_order_report_data_shipping

Reply:
Oh, yes! I build the schema based on records I got from the API... The schema Amazon shows is pretty bad. Good call I'll add some comments.